app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,37 @@
+class MenusController < ApplicationController
+  def index
+    @menus = Menu.all
+    authorize @menus
+  end
+
+  def new
+    if Date.today.saturday? || Date.today.sunday?
```

Comment:
- assign facade above
- define necessary methods in facade
- define all redirect cases with guard clauses, e.g. `redirect_to root_path, alert: t('menus.new.messages.weekend')`